barretenberg/cpp/src/barretenberg/stdlib/recursion/honk/verifier/protogalaxy_recursive_verifier.test.cpp

```diff
@@ -299,82 +294,69 @@ template <typename RecursiveFlavor> class ProtoGalaxyRecursiveTests : public tes
     static void test_tampered_decider_proof()
     {
         // Create two arbitrary circuits for the first round of folding
-        InnerBuilder builder1;
```

Comment:
The creation of these circuits and their instances has been moved to the `fold_and_verify_native`